confdb/aclocal_util.m4

```diff
@@ -79,12 +89,22 @@ dnl This is typically used for LIBS and similar variables because libraries
 dnl should be added in reverse order.
 AC_DEFUN([PAC_PREPEND_FLAG],[
         AC_REQUIRE([AC_PROG_FGREP])
-        AS_IF(
-                [echo "$$2" | $FGREP -e "$1" >/dev/null 2>&1],
-                [echo "$2(='$$2') contains '$1', not prepending" >&AS_MESSAGE_LOG_FD],
-                [echo "$2(='$$2') does not contain '$1', prepending" >&AS_MESSAGE_LOG_FD
-                $2="$1 $$2"]
-        )
+        AC_CACHE_CHECK([check if fgrep support -w option], ac_cv_path_FGREP_has_word,
+           [if echo 'ab*c' | $GREP -Fw 'ab*c' >/dev/null 2>&1
+           then ac_cv_path_FGREP_has_word="yes"
+           else ac_cv_path_FGREP_has_word="no"
+           fi])
```

Comment:
In stead of duplicating the code, how about 
```
AC_DEFUN([PAC_FGREP], ...)
```
which should accept old string, pattern string, true case, false case?

Then both `PAC_APPEND_FLAG` and `PAC_PREPEND_FLAG` can just call `PAC_FGREP`?

Reply:
Sounds good.